_data/members.yml

```diff
@@ -1,63 +1,98 @@
+  github: Gchism94
+  headshot: gregchism.jpg
   bio: 
-    I work as a scientific programmer for the Agricultural Data Science Team. My hobbies include contemplating the ways in which open data and open science can change the world, trying to guess people's Hogwarts House (Slytherin), and my cat.
-
-- name: Ailene Yee
-  affiliation: CyVerse
-  github: yeea
-  headshot: yeea.jpg
-  bio: 
-    I'm a software engineer with CyVerse who also studies the ancient Greco-Roman world. Making things accessible to others, reading, biking, eating and eating more, languages, and video games are my bread and butter.
-
-- name: Drake Asberry
-  affiliation: Second Language Acquisition and Teaching
-  github: drakeasberry
-  headshot: drakeasberry.png
-  bio: 
-    I am a PhD Candidate in Second Language Acquisition and Teaching (SLAT). You can find me at Public, the University, climbing big rocks, or in the Slytherin Common Room.
-
-- name: Rebecca Vanderpool
-  affiliation: College of Medicine
-  github: vanderpoolrr
-  headshot: vanderpoolrr.jpg
-  bio: 
-    I am an Assistant Professor in the College of Medicine. My research focuses on using CMR imaging and cardiopulmonary exercise testing to assess the pathophysiology of right ventricular function and to develop clinically useful biomarkers. 
-
-- name: Alex Bigelow
-  affiliation: Computer Science
-  github: alex-r-bigelow
-  headshot: alex-r-bigelow.jpg
-  bio: 
-    I'm a postdoc from Computer Science, studying how people think about, reshape, and visualize data. I build web-based tools to make these processes less painful, and more accessible to more diverse data workers. Slytherin.
-
-- name: Blake Joyce
-  affiliation: UITS Research Computing
-  github: bjoyce3
-  headshot: bjoyce3.jpg
-  bio:
-    I enjoy traveling, research computing, kendo, and deep-frying turkeys. Hufflepuff, of course.
 
 - name: Julian Pistorius
-  affiliation: Data Science and Infrastructure for Agriculture Group (DIAG)
+  affiliation: Exosphere Project
   github: julianpistorius
   headshot: julianpistorius.jpg
   bio:
     I build software and infrastructure for researchers. I create software for fun. I like to talk about software. I don't understand what "work-life balance" means. I <em>do</em> like to ride motorcycles and go hiking. Gryffindor.
 
-- name: Kelsey E Gonzalez
-  affiliation: Sociology
-  github: kelseygonzalez
-  headshot: kelseygonzalez.jpg
+- name: Alex Bigelow
+  affiliation: Department of Computer Science, UArizona
+  github: alex-r-bigelow
+  headshot: alex-r-bigelow.jpg
   bio: 
-    As a PhD student in Sociology, I am an advocate for data science literacy and love experimenting with R.  Besides learning new methodologies, I enjoy baking pizza and hiking on Mt Lemmon with my dog.
+    I'm a postdoc from Computer Science, studying how people think about, reshape, and visualize data. I build web-based tools to make these processes less painful, and more accessible to more diverse data workers. Slytherin.
```

Comment:
```suggestion
  affiliation: [Stardog Union](https://www.stardog.com)
  github: alex-r-bigelow
  headshot: alex-r-bigelow.jpg
  bio:
    I research how people think about, reshape, and visualize data—especially as network structures. I'm a Software Engineer on Stardog's Visual Experience Team, that builds visual tools to democratize these processes, making it easier for anyone to build knowledge graphs from data wherever it happens to live.
```

Reply:
I had to add this update manually, but it's in the PR.